src/main/java/org/broadinstitute/hellbender/engine/FeatureInput.java

```diff
@@ -267,6 +271,14 @@ public String getName() {
         return name;
     }
 
+    /**
+     *
+     * @return true if the value for name matches the default
```

Comment:
The javadoc for this method describes the inverse of what it does.

Reply:
done.